src/lexer/common.ts

```diff
@@ -28,26 +28,42 @@ export const enum NumberKind {
  */
 export function advanceChar(parser: ParserState): number {
   parser.column++;
-  return advanceAndLawrenceDolTheCRLF(parser);
-}
-
-export function advanceAndLawrenceDolTheCRLF(parser: ParserState) {
   parser.currentChar = parser.source.charCodeAt(++parser.index);
-  if (parser.index < parser.end) {
-    const cur = parser.currentChar;
-    const nxt = parser.source.charCodeAt(parser.index + 1);
-    if (
-      (cur == Chars.CarriageReturn && nxt == Chars.LineFeed) ||
-      (cur == Chars.LineFeed && nxt == Chars.CarriageReturn)
-    ) {
-      parser.currentChar = Chars.LineFeed;
-      parser.index++;
-    } else if (cur === Chars.CarriageReturn) {
-      parser.currentChar = Chars.LineFeed;
-    }
+
+  if(parser.index < parser.end) {
+      const cur = parser.currentChar;
+      const nxt = parser.source.charCodeAt(parser.index + 1);
+      if((cur == Chars.CarriageReturn && nxt == Chars.LineFeed) || (cur == Chars.LineFeed && nxt == Chars.CarriageReturn)) {
+          parser.currentChar = Chars.LineFeed;
+          parser.index++;
+      } else if(cur == Chars.CarriageReturn) {
+          parser.currentChar = Chars.LineFeed;
+      }
   }
   return parser.currentChar;
 }
+//export function advanceChar(parser: ParserState): number {
+//  parser.column++;
+//  return advanceAndLawrenceDolTheCRLF(parser);
+//}
+//
+//export function advanceAndLawrenceDolTheCRLF(parser: ParserState) {
+//  parser.currentChar = parser.source.charCodeAt(++parser.index);
+//  if (parser.index < parser.end) {
+//    const cur = parser.currentChar;
+//    const nxt = parser.source.charCodeAt(parser.index + 1);
+//    if (
+//      (cur == Chars.CarriageReturn && nxt == Chars.LineFeed) ||
+//      (cur == Chars.LineFeed && nxt == Chars.CarriageReturn)
+//    ) {
+//      parser.currentChar = Chars.LineFeed;
+//      parser.index++;
+//    } else if (cur === Chars.CarriageReturn) {
+//      parser.currentChar = Chars.LineFeed;
+//    }
+//  }
+//  return parser.currentChar;
+//}
```

Comment:
Just remove the code you commented out to make the source clean

Reply:
Yeah, I wasn't expect to be do a pull request so I left in the altered code to ease your checking.